File: frontend/lib/model/judgement/sentiment/command/DeleteSentimentAndChoiceJudgementCommand.ts
import IDeleteJudgementCommand from "../../command/IDeleteJudgementCommand";

export default class DeleteSentimentAndChoiceJudgementCommand implements IDeleteJudgementCommand {

    public readonly owner: string;
    public readonly project: string;
    public readonly phase: string;

    public readonly instance: string;
    public readonly annotator: string;

    public readonly UUID: string;

    constructor(owner: string, project: string, phase: string, instance: string, annotator: string, UUID: string) {
        this.owner = owner;
        this.project = project;
        this.phase = phase;

        this.instance = instance;
        this.annotator = annotator;

        this.UUID = UUID;
    }

}

File: backend/src/main/java/de/garrafao/phitag/application/judgement/sentimentandchoice/data/DeleteSentimentAndChoiceJudgementCommand.java
package de.garrafao.phitag.application.judgement.sentimentandchoice.data;

import de.garrafao.phitag.application.judgement.data.IDeleteJudgementCommand;
import lombok.Getter;

@Getter
public class DeleteSentimentAndChoiceJudgementCommand implements IDeleteJudgementCommand {

    private String owner;
    private String project;
    private String phase;

    private String instance;
    private String annotator;
    
    private String UUID;

    public DeleteSentimentAndChoiceJudgementCommand(String owner, String project, String phase, String instance, String annotator, String UUID) {
        this.owner = owner;
        this.project = project;
        this.phase = phase;

        this.instance = instance;
        this.annotator = annotator;
        
        this.UUID = UUID;
    }
    
}
